tests/Api/Admin/ProductAttributesTest.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Admin;
+
+use Sylius\Component\Attribute\AttributeType\SelectAttributeType;
+use Sylius\Component\Product\Model\ProductAttributeInterface;
+use Sylius\Tests\Api\JsonApiTestCase;
+use Sylius\Tests\Api\Utils\AdminUserLoginTrait;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ProductAttributesTest extends JsonApiTestCase
```

Comment:
We should add here also a test for adding a product attribute with a defined type and not valid content